scripting/include/executes.inc

```diff
@@ -27,6 +27,17 @@ enum StratType {
   StratType_ForceBuy = 2,
 };
 
+enum CTRiflePref {
+  CTRiflePref_M4 = 0,
+  CTRiflePref_Silenced_M4 = 1,
+  CTRiflePref_Aug = 2,
+}
+
+enum TRiflePref {
+  TRiflePref_Ak = 0,
+  TRiflePref_Sg = 1,
+}
```

Comment:
Move these somewhere else, anything in executes.inc is intended to be part of the public API.